recipe/meta.yaml

```diff
@@ -35,10 +35,13 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('m2w64_c') }}    # [win]
-    - {{ compiler('m2w64_cxx') }}  # [win]
-    - {{ compiler('cuda') }}       # [cuda_compiler != "None" and cuda_compiler_version != "None"]
+    - {{ compiler('cuda') }}           # [cuda_compiler != "None" and cuda_compiler_version != "None"]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ compiler('m2w64_fortran') }}  # [win]
```

Comment:
Do we still need MinGW Fortran or can it be dropped?

Reply:
This should probably dropped - the only reason it might be needed is if `cmake` checks for a Fortran compiler and fails if it doesn't find one.

Reply:
This is probably leftover from my past attempts to get things working ( https://github.com/conda-forge/xgboost-feedstock/commit/4ccf854fe2c91f69b379dbcdb215eb0035d82c09 ). Likely it is safe to drop